packages/block-library/src/post-time-to-read/block.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "core/post-time-to-read",
+	"title": "Post Time To Read",
+	"category": "theme",
+	"description": "Time to read the post.",
+	"textdomain": "default",
+	"usesContext": [ "postId", "postType" ],
+	"attributes": {
+		"textAlign": {
+			"type": "string"
+		},
+		"prefix": {
+			"type": "string",
+			"default": ""
+		},
+		"suffix": {
+			"type": "string",
+			"default": ""
+		}
+	},
+	"supports": {
+		"html": false,
+		"multiple": false
+	}
```

Comment:
As a follow-up PR once this is merged, it'd be good to consider what design tools this needs. There are some tracking issues for design tool consistency - https://github.com/WordPress/gutenberg/issues?q=is%3Aopen+is%3Aissue+label%3A%22%5BFeature%5D+Design+Tools%22+label%3A%22%5BType%5D+Tracking+Issue%22.